Dockerfile

```diff
@@ -14,7 +16,10 @@ ENV ME_CONFIG_EDITORTHEME="default" \
 
 ENV MONGO_EXPRESS 0.51.0
 
-RUN npm install mongo-express@$MONGO_EXPRESS
+RUN set -eux; \
+	apk add --no-cache --virtual .me-install-deps git; \
```

Comment:
> ... this pre-emptively installs `git` temporarily during `npm install` since 0.54.0 has a git-based dependency.

I did a build test with 0.54.0 (without committing the bump) and this was necessary there so I'm pre-adding that so the bump goes without a hitch. :innocent: